docs/contribute/contribution_guidelines.rst

```diff
@@ -70,6 +70,15 @@ The testing for functions and classes should at least contain tests for
 
 In case of deprecations, it must also be tested if the deprecated feature is properly removed (see `tests/test_deprecations.py` for examples).
 
+Testing plotting functions
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The above also holds for plotting functions, however, in this case it is required that the plots generate baseline and test plots. Baseline plots are visually inspected when they are first created and serve as a reference. Test plots are created each time the test are ran. If a pull request changes a plot function, the test plots are visually compared against the baseline plots.
```

Comment:
maybe add the case for new plots, like creating baselines

Reply:
I tried to make this more clear and now call them baseline and test images instead of plots. This should better distinguish between plot functions and the images that they produce.